src/Fable.Transforms/Python/Fable2Python.fs

```diff
@@ -548,17 +653,22 @@ module Helpers =
         let moduleName =
             let lower =
                 let fileName = Path.GetFileNameWithoutExtension(modulePath)
+
                 match fileName with
                 | "" when modulePath.StartsWith(".") -> modulePath.[1..]
-                | _ -> fileName |> Naming.applyCaseRule CaseRules.SnakeCase
-            (lower, Naming.NoMemberPart) ||> Naming.sanitizeIdent (fun _ -> false)
+                | _ ->
+                    fileName
+                    |> Naming.applyCaseRule CaseRules.SnakeCase
+
+            (lower, Naming.NoMemberPart)
+            ||> Naming.sanitizeIdent (fun _ -> false)
 
         let path =
             match relative, normalizedPath with
             | _, "" -> ""
             | true, "." -> "."
             | true, path ->
-                path  // translate path to Python relative syntax
+                path
```

Comment:
Fantomas has removed the comment here.

Reply:
Yes, I've seen this happen a couple of times. Added issue at: https://github.com/fsprojects/fantomas/issues/2027